src/JMS/Serializer/XmlSerializationVisitor.php

```diff
@@ -454,7 +454,7 @@ private function addNamespaceAttributes(ClassMetadata $metadata, \DOMElement $el
 
     private function createElement($tagName, $namespace = null)
     {
-        if (null === $namespace) {
+        if (null === $namespace || $this->document->isDefaultNamespace($namespace)) {
```

Comment:
this is not always true. 

elements should be created in their namespace, if they belong to one. then the dom parser will take care of converting it into the correct xml reppresentation